Sources/StreamVideo/WebRTC/PeerConnectionFactory.swift

```diff
@@ -20,7 +20,7 @@ final class PeerConnectionFactory: @unchecked Sendable {
         )
         let decoderFactory = RTCDefaultVideoDecoderFactory()
         return RTCPeerConnectionFactory(
-            audioDeviceModuleType: .platformDefault,
+            audioDeviceModuleType: .audioEngine,
```

Comment:
why do we change this here?

Reply:
The platformDefault ADM causes the prompts to appear while the audioEngine based one doesn't. We are already planning on migrating to the AudioEngine ADM so we can also do it here and resolve the issue.